chore: precommit-guard-tests workflow の期待 SKIP count drift を修正 (12 → 11)

## 概要

複数 PR (#595, #596, #597, #598, #599) で `precommit-guard-tests` ジョブが共通失敗している。原因は workflow の assert ステップが `SKIP(known-limitation): 12` を期待するが、harness 出力は `SKIP(known-limitation): 11` となっている数値 drift。

## 失敗ログ抜粋

\`\`\`
##[error]Expected 'SKIP(known-limitation): 12' but harness output diverged.
TOTAL: 27  OK: 16  FAIL: 0  SKIP(known-limitation): 11
\`\`\`

## 想定原因

直近 master commit \`2ec3e34\` (PR #594 「chore: pre-commit-guard.sh の設計方針確定 + R06 期待値修正 (Issue #568 設計合意フェーズ)」) で harness 側の R06 期待値が 1 件修正された一方、CI workflow 側の assert 値 (12) が同期更新されなかった可能性。

## 対応案

- harness 出力 \`TOTAL: 27 / OK: 16 / SKIP: 11\` が正しい場合: workflow の assert を 11 へ更新
- assert 12 が正しい場合: harness 側に取りこぼし R が存在する。harness を修正
- 一次調査で「どちらが正か」を確定してから片側のみ更新する

## 影響

- 現状、複数 chore PR (#595-#599) が CI 上で UNSTABLE になり merge bot やレビュー判定が混乱する
- 既存 master HEAD でも同じ fail が出る既存課題のため、master 起点の他 PR にも影響する

## 受け入れ基準

- [ ] master 単独で precommit-guard-tests が pass する
- [ ] PR #595-#599 でも precommit-guard-tests が pass する (rebase / merge 不要、原因のみが共有)
- [ ] どちらを正としたかの判断根拠を PR 本文に明記

## 関連

- 関連 PR: #594 (drift 導入の疑い), #587 (CI 統合元)
- 関連 Issue: #568, #592

--- a/.github/workflows/precommit-guard-tests.yml
+++ b/.github/workflows/precommit-guard-tests.yml
@@ -90,7 +90,7 @@ jobs:
             echo "## pre-commit-guard test harness";
             echo "";
             if [ -n "$out_file" ] && [ -f "$out_file" ]; then
-              # TOTAL 行 (例: "TOTAL: 27  OK: 15  FAIL: 0  SKIP(known-limitation): 12")
+              # TOTAL 行 (例: "TOTAL: 27  OK: 16  FAIL: 0  SKIP(known-limitation): 11")
               # を抽出して目立つ位置に置く。grep が一致しない場合に備えて || true。
               total_line=$(grep -E '^TOTAL:' "$out_file" || true)
               if [ -n "$total_line" ]; then

--- a/.github/workflows/precommit-guard-tests.yml
+++ b/.github/workflows/precommit-guard-tests.yml
@@ -56,10 +56,12 @@ jobs:
         # literal 一致 grep で固定化する。
         #
         # harness の TOTAL 行例:
-        #   TOTAL: 27  OK: 15  FAIL: 0  SKIP(known-limitation): 12
+        #   TOTAL: 27  OK: 16  FAIL: 0  SKIP(known-limitation): 11
         #
-        # 期待値は現状 12 (Issue #550 の DA レビューで指摘された既存 bypass 11
-        # + R06 nested subshell 1)。新たに known-limitation が追加された
+        # 期待値は現状 11 (Issue #568 / PR #594 で R06 nested subshell を
+        # known-limitation から「block」期待値へ昇格させたことで、Issue #550
+        # の DA レビューで指摘された既存 bypass 11 件 = E01-E11 のみが
+        # known-limitation として残る)。新たに known-limitation が追加された
         # (= SKIP が増えた) regression を CI 上で検知し、harness 側と本
         # workflow 側を同 PR で更新する運用に倒す。
         run: |
@@ -69,8 +71,8 @@ jobs:
             echo "::error::harness output file not found: $out_file"
             exit 1
           fi
-          if ! grep -qE 'SKIP\(known-limitation\): 12$' "$out_file"; then
-            echo "::error::Expected 'SKIP(known-limitation): 12' but harness output diverged."
+          if ! grep -qE 'SKIP\(known-limitation\): 11$' "$out_file"; then
+            echo "::error::Expected 'SKIP(known-limitation): 11' but harness output diverged."
             echo "::error::Check harness output below and update either the harness or this workflow in the same PR."
             grep -E '^TOTAL:' "$out_file" || echo "(TOTAL line not found in harness output)"
             exit 1

